src/cirrus/gitconfig.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+"""
+gitconfig
+
+local gitconfig parser/updater since the lib on pypi seems to be busted
+
+"""
+import os
+import operator
+import subprocess
+import contextlib
+
+
+@contextlib.contextmanager
+def gitconfig(filename="~/.gitconfig"):
+    c = GitConfig(filename)
+    c.parse()
+    yield c
```

Comment:
Any reason you use this as a context manager when nothing happens after the yield?